quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMappers.java

```diff
@@ -257,8 +257,6 @@ private void remove(String key, PropertyMapper<?> mapper) {
         }
 
         public void sanitizeDisabledMappers() {
-            if (Environment.getParsedCommand().isEmpty()) return; // do not sanitize when no command is present
```

Comment:
@vmuzikar AFAIR, it was used for cases when no command is provided and help msg should be shown. I don't think it's gonna work. The parsed command should be set even for the fast startup (`kc.sh start --optimized`) - the `start` is the parsed command. And even for the startup, the sanitization should normally work - https://github.com/keycloak/keycloak/blob/main/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/KeycloakMain.java#L83.

There might be some more deeply hidden issue - I can take a look at it tomorrow :)

Reply:
> It was not as we initialize the commands only when Picocli is initialized too. I added an explicit command set for the fast startup. Let's see if the tests pass.

Yes, you're definitely right.